packages/cloud-toolkit/src/services/http-forest-server.ts

```diff
@@ -10,7 +10,8 @@ async function handledAxios<T>(
   try {
     const response = await axios(axiosRequestConfig);
 
-    if (response.status !== 200) {
+    // 2xx http codes = no error
+    if (Math.floor(response.status / 100) !== 2) {
```

Comment:
```suggestion
    if (response.status < 400) {
```

This will be clearer and will handle possible 300 responses, if we do implement a 304 like previously discussed